v22.2/restore.md

```diff
@@ -28,12 +28,47 @@ You can restore:
 
 ## Required privileges
 
+{{site.data.alerts.callout_info}}
+From v22.2, CockroachDB supports a restore privilege model that provides finer control over a user's privilege to restore a backup. We recommend implementing this privilege model for restores.
+
+There is continued support for the [existing privilege model](#existing-required-privileges) in v22.2. However, this will be removed in a future release.
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v22.2" %} You can grant the `RESTORE` privilege to a user or role depending on the type of restore required:
+
+Restore | Privilege
+-------+-----------
+Cluster | Grant a user the system level `RESTORE` privilege.
+Database | Grant a user the system level `RESTORE` privilege to restore databases onto the cluster.
+Table | Grant a user the database level `RESTORE` privilege to restore schema objects into the database.
+
+The listed privileges do not cascade to objects lower in the schema tree. For example, if you are granted system-level restore privileges, this does not give you the privilege to restore a table. If you need the `RESTORE` privilege on a database to apply to all newly created tables in that database, use [`DEFAULT PRIVILEGES`](security-reference/authorization.html#default-privileges).
```

Comment:
Would they know how to use Default privileges.. that links to a very generic page. Would adding something like --> Add BACKUP to the user/role's default privileges make sense?